src/main/cpp/Monopoly.cpp

```diff
@@ -19,46 +20,50 @@ bool Monopoly::gameOver() {
 	return (playersInGame <= 1);
 }
 
-int Monopoly::roll() {
+int 
+Monopoly::roll() {
     dice1 = rand() % 6 + 1;
 	dice2 = rand() % 6 + 1;
     
     return dice1 + dice2;
 }
 
-bool Monopoly::rollDoubles() {
+bool 
+Monopoly::rollDoubles() {
     return dice1 == dice2;
 }
 
-int Monopoly::getRoll() {
+int 
+Monopoly::getRoll() {
     return dice1 + dice2;
 }
 
-int Monopoly::getDie(int v) {
+int 
+Monopoly::getDie(int v) {
 	return v == 1 ? dice1 : dice2;
 }
 
-void Monopoly::attemptOutOfJail() {
+void 
+Monopoly::attemptOutOfJail() {
     
-    Player *player = getCurrentPlayer();
+    shared_ptr<Player> player(getCurrentPlayer());
     int turnsInJail = player->getTurnsInJail();
 
-    cout << "You have been in jail for " << turnsInJail << " turns." << endl;
-    getchar();
+	print("You have been in jail for " + to_string(turnsInJail) + " turns.", true, false);
     if(turnsInJail < 3) {
         if(player->getMoney() >= 50) {
             
             string ans;
             bool answered = false;
 
             while (answered) {
-                cout << "Would you like to get out of jail for $50? (y/n)" << endl;
+				print("Would you like to get out of jail for $50? (y/n)", false, false);
```

Comment:
You can ignore the print functions. I added the commit for this into the main branch.

Reply:
I don't think I messed with this, it may be part of the rebase?